### DISCUSSION TITLE
Images showing in markdown preview but not website (GitHub pages)?

### DISCUSSION THREAD

**Discussion by cconn30** [community/community#74768] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I cannot get images to show up on my GitHub pages site, however they do show up on the remote repository in the markdown preview. I had all of my images in `static`, however the only one which did indeed show on the website was an image which is referenced in `_index.md`. I moved the remaining images (and PDFs) to `assets` as I saw recommended on another community post, but to no avail.

I have checked spelling/capitalization, about every filepath that I can think of, tried both HTML and markdown, and every other solution I've seen on the internet. I can't figure this out, help would be massively appreciated! Thank you!

Note: the images which do not work are in /content/post/photogallery.md

repo: https://github.com/cconn30/ccxyz

**Comment by mutasim77** [comment]
do not import images from the root ```"/"```, but  the exact path to that image: 
```html
<img src="../../assets/dm1.jpg" alt="dartmouth 1" width="225">
....
```

**Reply by cconn30** [reply]
I just tried this, and unfortunately the same issue persists. The markdown preview shows the image, but it still doesn't show on the website. Could there be some issue with my configuration file `hugo.toml`? I've seen some other things online which referenced issues with that, but I can't find anything in particular which could be wrong with mine. I appreciate the fast response though!

**Comment by ghost** [comment, marked as answer]
hi i made a pull request see changes i made it is showing all images and files
https://lisztomania23.github.io/ccxyz/post/photogallery/
though i haven't worked with hugo so can't tell what was causing the problem or how to fix it

**Reply by cconn30** [reply]
Thank you so much! This worked.

**Reply by ghost** [reply]
your website looking good i'm gonna make some static website now
